sdk/python/feast/infra/offline_stores/snowflake.py

```diff
@@ -152,6 +155,21 @@ def pull_latest_from_table_or_query(
             + '"'
         )
 
+        if config.offline_store.convert_timestamp_columns:
+            select_timestamps = list(map(lambda field_name: f"to_varchar({field_name}, 'YYYY-MM-DD\"T\"HH24:MI:SS.FFTZH:TZM') as {field_name}", timestamp_columns))
+            inner_field_string = (
```

Comment:
Can you use an f string here? Maybe combined with a multi line string (using triple quotes)? Just for readability